src/project/qml/MuseScore/Project/internal/SaveToCloud/SaveLocationOption.qml

```diff
@@ -92,6 +94,7 @@ ColumnLayout {
             FlatButton {
                 id: button
                 accentButton: true
+
```

Comment:
When navigating through the buttons, the user should hear full information about button's section.
button.accessible.name = root.title + " " + root.description + " " button.text

Reply:
I put the description in `accessible.description`. Wouldn't that be appropriate?